# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24-psionw/05-devfs-not-exp.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Currently (at version 2.0.x) ROCK Linux required DevFS. It is highly
useful. Later version of ROCK Linux are planed to support other variants
(like old-style device nodes or udevfs).

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.23/fs/Config.in.orig	2003-11-28 19:26:21.000000000 +0100
+++ linux-2.4.23/fs/Config.in	2003-12-09 00:55:11.000000000 +0100
@@ -72,13 +72,9 @@
 
 bool '/proc file system support' CONFIG_PROC_FS
 
-# For some reason devfs corrupts memory badly on x86-64. Disable it 
-# for now.
-if [ "$CONFIG_X86_64" != "y" ] ; then
-dep_bool '/dev file system support (EXPERIMENTAL)' CONFIG_DEVFS_FS $CONFIG_EXPERIMENTAL
+dep_bool '/dev file system support (Required for ROCK Linux)' CONFIG_DEVFS_FS $CONFIG_EXPERIMENTAL
 dep_bool '  Automatically mount at boot' CONFIG_DEVFS_MOUNT $CONFIG_DEVFS_FS
 dep_bool '  Debug devfs' CONFIG_DEVFS_DEBUG $CONFIG_DEVFS_FS
-fi
 
 # It compiles as a module for testing only.  It should not be used
 # as a module in general.  If we make this "tristate", a bunch of people
